--- ./src/modules/about/aboutdialog.cpp.orig	2006-07-25 13:02:18.000000000 +0000
+++ ./src/modules/about/aboutdialog.cpp	2006-07-25 13:03:01.000000000 +0000
@@ -129,7 +129,7 @@
 	QString szLicense;
 
 	QString szLicensePath;
-	g_pApp->getGlobalKvircDirectory(szLicensePath,KviApp::License,"COPYING");
+	g_pApp->getGlobalKvircDirectory(szLicensePath,KviApp::License,"EULA");
 	
 	if(!KviFileUtils::loadFile(szLicensePath,szLicense))
 	{
--- ./src/modules/setup/setupwizard.cpp.orig	2006-07-25 13:02:29.000000000 +0000
+++ ./src/modules/setup/setupwizard.cpp	2006-07-25 13:03:07.000000000 +0000
@@ -161,7 +161,7 @@
 	ed->setWordWrap(QTextEdit::NoWrap);
 	KviStr szLicense;
 	KviStr szLicensePath;
-	g_pApp->getGlobalKvircDirectory(szLicensePath,KviApp::License,"COPYING");
+	g_pApp->getGlobalKvircDirectory(szLicensePath,KviApp::License,"EULA");
 	QString buf;
 	if(!KviFileUtils::loadFile(szLicensePath.ptr(),buf))
 	{
